.ci/errors-ascii-chars

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+echo "Checking for non-ASCII characters in error messages"
+echo "These messages end up in places with limitations like VCL"
+negated_ascii_range="[^\x00-\x7F]"
+matching=$(grep \
+    --color='auto' \
+    --perl-regexp "$negated_ascii_range" \
+    --line-number \
+    --recursive \
+    source/_patterns/00-atoms/errors/ || true)
+if [ "$matching" != "" ]; then
+    echo "Non-ASCII characters found:"
```

Comment:
Could this be expanded to state which file's contents generated has the error?

Reply:
It does state this:
```
$ .ci/errors-ascii-chars 
Checking for non-ASCII characters in error messages
These messages end up in places with limitations like VCL
Non-ASCII characters found:
source/_patterns/00-atoms/errors/server-error.mustache:13:    <p>Something has gone seriously wrong.<br>We’ve logged the problem and will look into it.</p>
```

Reply:
I have forced colors to be enabled:
<img src="https://screenshots.firefox.com/m7K0MG96JRmOpF73/alfred.elifesciences.org" width="500" />